lib/common/acl.c

```diff
@@ -327,6 +327,22 @@ pcmk__unpack_acl(xmlNode *source, xmlNode *target, const char *user)
     }
 }
 
+/*!
+ * \internal
+ * \brief Copy source to target and set xf_acl_enabled flag in target
+ *
+ * \param[in]     acl_source    XML with ACL definitions
+ * \param[in,out] target        XML that ACLs will be applied to
+ * \param[in]     user          Username whose ACLs need to be set
+ */
+void
+pcmk__set_xf_acl_enabled(xmlNode *acl_source, xmlNode *target, const char *user)
```

Comment:
This does more than set the flag, so let's name it more clearly, something like pcmk__unpack_and_apply_acl() or pcmk__enable_acl()